scripts/settings.lua

```diff
@@ -236,7 +235,7 @@ function BCsetting.new(data, customMt)
 	self.type = AIParameterType.SELECTOR
 	self.name = data.name
 	self.data = data
-	if next(data.values) ~=nil then
+	if data.values ~=nil and next(data.values) ~=nil then
```

Comment:
I had to add this nil check since next(nil) throws, but other than that the code which expands min/max etc works